### DISCUSSION TITLE
Beidou APC corrections

### DISCUSSION THREAD

**Discussion by k9891** [groops-devs/groops#179] [Q&A]
I have some questions about BDS APC corrections. In receiver and satellite antenna corrections, there is an option called "use nearest frequency," , What does this mean？ Is this for the frequency of all GNSS, or is it modified using GPS L1/L2 as a traditional approach.

For example, the satellite PCO corrections on BDS-3 B2a frequency are not available, Which frequency is considered the same for PCO correction on B2a frequency？ Meanwhile, since the receiver PCO/PCV  for BDS were still unavailable at most of stations, is it assumed that the calibration of BDS receiver PCO/PCV is the same as that of GPS L1/L2, or GLONASS?

Thank in advance for your time and help.
I am looking forward to hearing from you.

With kind regards,
k9891

**Comment by tmayerguerr** [comment]
Hi k9891,

I don't know the traditional approach (modified using GPS L1/L2?). Can you please explain it to me?
If PCO corrections on a specific [GnssType](https://groops-devs.github.io/groops/html/gnssType.html) are not available it used the first one in the antenna list with the closest frequency difference. E.g. for BDS-3 B2a it uses the corrections for GPS L5 instead.

regards
Torsten

**Reply by k9891** [reply]
Hello Tosten,

Generally, only a few antenna types could provide Beidou-3 PCO/PCV corrections in the IGS14.atx file. Therefore, we simply used the GPS L1 corrections to approximate Beidou-3 B1I corrections, while all other frequencies shared the same PCO/PCV as those of GPS L2.
While, In the satellite-end, Does GROOPS use the closest frequency in the antenna list of the satellite to replace it, such as which BDS B2 (1191.795) uses, B2b (1207.14), or B2a (1176.45),or other gnsstypes？

With kind regards,
 K9891
